### ISSUE TITLE
BKVGG12 Architecture

### ISSUE THREAD

**Issue by krishy91** [cvasfi/dl4cv#2]
Need to have working BKVGG12 network as described in 'Facial Expression Recognition Using Deep
Convolutional Neural Networks' by Dinh Viet Sang

Working means we should be able to overfit the network on small sample from the dataset & even for the full dataset, accuracy should be greater than random selection and ideally close to value achieved value in the paper (71.6)
